sproutsocial/seeds-packets
# [BCT-3527] Dependency upgrade

## Description:

* Updates outdated dependencies to resolve github security warnings

### How To Validate

1. Pull down `dependency-upgrade` branch
1. Run `yarn` and `yarn build`
1. See there's no issues
